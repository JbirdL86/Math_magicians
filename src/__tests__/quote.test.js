import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Quote from '../components/quote';

it('renders correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});

describe('Quote component', () => {
  test("Rendererd component contains the word 'Mathematics'", () => {
    render(<Quote />);
    expect(screen.getByText(/Mathematics/i)).toBeInTheDocument();
  });
});
