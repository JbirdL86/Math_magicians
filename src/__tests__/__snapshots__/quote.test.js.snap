// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quote_container"
>
  <p
    className="main_quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston.
  </p>
</div>
`;
